edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+  <h4> {% trans "See Also" %} </h4>
+    <p>
+    {% blocktrans %}
+      <a class="footer-link" href="#">
+        Frequently asked questions about proctoring and earning college credit.
```

Comment:
Delete the period at the end.
Link can go to: https://www.edx.org/about/student-faq